service/src/test/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderServiceTest.java

```diff
@@ -245,5 +319,11 @@ private CaseData populatedCaseData(LocalDate localDate) {
                     .build())
                 .build()))
             .build();
+
+        return caseDataBuilder.build();
+    }
+
+    private String formatTypeToFileName(String type) {
+        return type.toLowerCase().replaceAll("[()]", "").replaceAll("[ ]", "_") + ".pdf";
     }
```

Comment:
In the test would be fine to hardcode the value which we expect - this hsa got some replacement logic which can be wrong!